src/azure-cli/azure/cli/command_modules/acs/azuremonitormetrics/recordingrules/delete.py

```diff
@@ -23,27 +23,28 @@ def delete_rule(cmd, cluster_subscription, cluster_resource_group_name, default_
 
 
 def delete_rules(cmd, cluster_subscription, cluster_resource_group_name, cluster_name):
+    # limit rule group name to 260 characters
     delete_rule(
         cmd,
         cluster_subscription,
         cluster_resource_group_name,
-        "NodeRecordingRulesRuleGroup-{0}".format(cluster_name)
+        "NodeRecordingRulesRuleGroup-{0}".format(cluster_name[:232])
```

Comment:
use a helper function for truncation?

Reply:
https://github.com/Azure/azure-cli/pull/27788/files